components/carousel/carousel.css

```diff
@@ -0,0 +1,83 @@
+.ui-carousel {
+  overflow: hidden;
+  position: relative;
+}
+
+.ui-carousel-item {
```

Comment:
I agree with @reaktivo that naming could be done better, but for me, it is not critical since components are quite isolated